### PR TITLE
Usman/lp 1750

### PR DESCRIPTION
**Pull request by muak50**
Make specialization certificate downloadable
https://philanthropyu.atlassian.net/browse/LP-1750

### CODE REVIEW

**Review by ifaizanahmedk — APPROVED** [review]

File: credentials/apps/credentials_theme_philu/static/sass/certificate-ltr.scss [hunk 1]
```diff
@@ -301,79 +301,80 @@ body {
...
+				display: flex;
+				align-items: center;
+				padding: 30px 10px 10px !important;
+				justify-content: center;
+				flex-direction: column;
+				background-color: white;
+				border: 15px solid #61B4E4;
+
+				&::before {
+					content: url('https://static.philanthropyu.org/philanthropyu-icon-black.svg');
```
Comment (emraan004) [inline credentials/apps/credentials_theme_philu/static/sass/certificate-ltr.scss]: Please use constant of static URL

File: credentials/apps/credentials_theme_philu/static/sass/certificate-ltr.scss [hunk 2]
```diff
@@ -301,79 +301,80 @@ body {
...
 				position: relative;
-
-				.certificate-stamp {
+				display: flex;
+				align-items: center;
+				padding: 30px 10px 10px !important;
+				justify-content: center;
+				flex-direction: column;
+				background-color: white;
+				border: 15px solid #61B4E4;
```
Comment (emraan004) [inline credentials/apps/credentials_theme_philu/static/sass/certificate-ltr.scss]: please use color variable

File: credentials/apps/credentials_theme_philu/static/sass/certificate-ltr.scss [hunk 3]
```diff
@@ -301,79 +301,80 @@ body {
...
-					@media (min-width: 992px) {
-						top: 70px;
-						left: 50px;
-					}
+					left: auto;
+					right: 0;
+					top: 0;
+					bottom: 0;
+					margin: auto;
+					background-color: white;
```
Comment (emraan004) [inline credentials/apps/credentials_theme_philu/static/sass/certificate-ltr.scss:336]: please use color variable

**Review by emraan004 — CHANGES_REQUESTED** [review]

File: credentials/apps/credentials_theme_philu/static/sass/certificate-ltr.scss (hunk 3 above)
Comment (muak50) [reply credentials/apps/credentials_theme_philu/static/sass/certificate-ltr.scss:336]: It's already core variable

**Review by HassanArif-arbisoft — APPROVED** [review]
LGTM 👍

**Review by emraan004 — APPROVED** [review]
👍

**Pull request merged**